packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -1276,11 +1338,11 @@ export = {
 
         expect(stack).to(haveResourceLike('AWS::CodeBuild::Project', {
           'Artifacts':
-            {
```

Comment:
If you're changing this, can you move the `{` to the upper line (`'Artifacts':`)?